From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sun, 27 Mar 2016 11:26:13 +0200
Subject: clk: unconditionally recurse into clk/mvebu/
Git-commit: a454018505f2e875441603b18dbbb912d422f7d1
Patch-mainline: v4.7-rc1
References: fate#319899 fate#320700

The drivers/clk/mvebu directory is only being built when
CONFIG_PLAT_ORION=y. As we are going to support additional mvebu
platforms in drivers/clk/mvebu, which don't have CONFIG_PLAT_ORION=y,
we need to recurse into this directory regardless of the value of
CONFIG_PLAT_ORION.

Since all files in drivers/clk/mvebu/ are already conditionally
compiled depending on various Kconfig options, we can recurse
unconditionally into drivers/clk/mvebu without any other change.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Signed-off-by: Stephen Boyd <sboyd@codeaurora.org>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/clk/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/clk/Makefile
+++ b/drivers/clk/Makefile
@@ -59,7 +59,7 @@ obj-$(CONFIG_ARCH_MEDIATEK)		+= mediatek
 ifeq ($(CONFIG_COMMON_CLK), y)
 obj-$(CONFIG_ARCH_MMP)			+= mmp/
 endif
-obj-$(CONFIG_PLAT_ORION)		+= mvebu/
+obj-y					+= mvebu/
 obj-$(CONFIG_ARCH_MESON)		+= meson/
 obj-$(CONFIG_ARCH_MXS)			+= mxs/
 obj-$(CONFIG_ARCH_LPC18XX)		+= nxp/
